src/main.ts

```diff
@@ -181,6 +182,29 @@ export default function main() {
     return res;
   }
 
+  function collectFeedback(options: FeedbackDialogOptions) {
```

Comment:
The SDK is used by both node and browser. But this method doesn't make sense anywhere else than a browser.

Potential solutions:
- Throw an explicit error in node (calling the function will already throw in node, but not with a good error message)
- Separate builds with separate API's for node and browser

I think I prefer throwing for now, to avoid getting into all sorts of dual configuration, deployment and documentation work